src/common/com/microsoftopentechnologies/intellij/helpers/UIHelper.java

```diff
@@ -0,0 +1,23 @@
+package com.microsoftopentechnologies.intellij.helpers;
+
+import javax.swing.*;
+
+public interface UIHelper {
+    void packAndCenterJDialog(JDialog form);
```

Comment:
There are 23 references to packAndCenterJDialog, all from IntelliJ specific classes. This method is specific to Swing dependent implementations (IntelliJ) and it shouldn't be exposed as part of this interface. I would leave it public in UIHelperImpl and call it directly from IntelliJ specific classes.